packages/Interface/src/DTK_UserDataInterface.hpp

```diff
@@ -106,41 +107,21 @@ using PolyhedronListDataFunction = std::function<void(
 
 //---------------------------------------------------------------------------//
 /*!
- * \brief Get the size parameters for building a cell list with a single
- * topology.
+ * \brief Get the size parameters for building a cell list.
  */
 using CellListSizeFunction =
     std::function<void( std::shared_ptr<void> user_data, unsigned &space_dim,
                         size_t &local_num_nodes, size_t &local_num_cells,
-                        unsigned &nodes_per_cell, bool &has_ghosts )>;
-
-//---------------------------------------------------------------------------//
-/*!
- * \brief Get the data for a single topology cell list.
- */
-using CellListDataFunction =
-    std::function<void( std::shared_ptr<void> user_data,
-                        View<Coordinate> coordinates, View<LocalOrdinal> cells,
-                        View<bool> is_ghost_cell, std::string &cell_topology )>;
-
-//---------------------------------------------------------------------------//
-/*!
- * \brief Get the size parameters for building a cell list with mixed
- * topologies.
- */
-using MixedTopologyCellListSizeFunction =
-    std::function<void( std::shared_ptr<void> user_data, unsigned &space_dim,
-                        size_t &local_num_nodes, size_t &local_num_cells,
                         size_t &total_nodes_per_cell, bool &has_ghosts )>;
```

Comment:
rename `total_nodes_per_cell` to `total_cell_nodes` or something along those lines